### PR TITLE
Move settings for /tmp to /etc/fstab so it works on all init systems.

### PR DESCRIPTION
**Pull request by diederikdehaas**

### CODE REVIEW

**Comment by Mausy5043** [comment]
Surprised to see you apparently only need `nodev` and `nosuid`.
:-1:

File: scripts/etc/init.d/rcS [hunk 1]
```diff
@@ -496,11 +496,8 @@ elif [ "$rootfstype" = "btrfs" ]; then
 else
 	echo "$rootpartition / $rootfstype $rootfs_mount_options 0 1" >> /rootfs/etc/fstab || fail
 fi
-echo "OK"
-
-# use ram tmpfs by default
-echo -n "  Configuring tmpfs to use RAM... "
-echo "RAMTMP=yes" >> /rootfs/etc/default/tmpfs 
+# also specify /tmp on tmpfs in /etc/fstab so it works across init systems
+echo "tmpfs /tmp tmpfs nodev,nosuid 0 0" >> /rootfs/etc/fstab || fail
```
Comment (goranche) [inline scripts/etc/init.d/rcS]: adding noexec might make sense as well

**Comment by goranche** [comment]
> Also no size-restriction?

I would leave that to the user

**Comment by goranche** [comment]
> Surprised to see you apparently only need nodev and nosuid.

@Mausy5043 what surprises you about that? mount has defaults for most options anyway, so if you're ok with those, you don't need to list them specifically :wink:

**Comment by diederikdehaas** [comment]
> Also no size-restriction?

I agree with @goranche to leave it up to the user. 
While it normally sets the size of tmpfs to 50% of RAM, it's not actually a problem, since [it's actual size is dynamically allocated according to the amount of data on /tmp](http://wiki.centos.org/TipsAndTricks/TmpOnTmpfs#head-9234b63540528fc2162ae32bf5a6939bbe43ae5d).

> adding noexec might make sense as well

I thought about that, but decided against it. See also the link above

**Pull request merged**